components/Analytics/PerformanceDashboard.tsx

```
@@ -0,0 +1,213 @@
+    }
+    return `${Math.round(value)}ms`;
+  };
+
+  return (
+    <div className={`fixed ${positionClasses[position]} z-50`}>
+      {!isVisible ? (
+        <button
+          onClick={() => setIsVisible(true)}
+          className="bg-blue-600 hover:bg-blue-700 text-white px-3 py-2 rounded-lg shadow-lg text-sm font-medium transition-colors"
+          title="Show Performance Dashboard"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)